src/main/scala/com/typesafe/conductr/sbt/ConductRPlugin.scala

```diff
@@ -45,6 +45,9 @@ object ConductRPlugin extends AutoPlugin {
       commands ++= Seq(controlServer),
       conduct := conductTask.value.evaluated,
       conductrDiscoveredDist <<= (dist in Bundle).storeAs(conductrDiscoveredDist in Global).triggeredBy(dist in Bundle),
+      conductrDiscoveredConfigDist <<=
+        (dist in BundleConfiguration).storeAs(conductrDiscoveredConfigDist in Global)
+        .triggeredBy(dist in BundleConfiguration),
```

Comment:
Style: could you make the `conductrDiscoveredDist` also span multiple lines for consistency?